ajax/libs/fonticonpicker/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "fonticonpicker",
+  "npmName": "fonticonpicker",
+  "version": "2.0.0",
+  "description": "An icon picker built on top of font icons and jQuery",
+  "keywords": [
+    "icons",
+    "font icons",
+    "picker"
+  ],
+  "filename": "jquery.fonticonpicker.min.js",
+  "homepage": "https://codeb.it/fonticonpicker/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/micc83/fontIconPicker.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git@github.com:micc83/fontIconPicker.git",
+    "fileMap": [
```

Comment:
Let's modify the auto-update config part as the followings:
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/micc83/fontIconPicker.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "jquery.fonticonpicker*",
          "css/**/*",
          "themes/**/*"
        ]
      }
    ]
  }
```

Because 1) target is the repo link which can be accessed. 2) basePath should be root bacause there is no release folder 3) files is the pattern which can be added by bot.

Many thanks.